the doctest of print_square function:

import the function:

>>> print_square = __import__('4-print_square').print_square

test with random numbers:

>>> print_square(3)
###
###
###

>>> print_square(5)
#####
#####
#####
#####
#####

>>> print_square(0)

test raising exceptions:

>>> print_square(3.0)
Traceback (most recent call last):
	...
TypeError: size must be an integer

>>> print_square("30")
Traceback (most recent call last):
	...
TypeError: size must be an integer

>>> print_square(-4)
Traceback (most recent call last):
	...
ValueError: size must be >= 0
